scripts/source-maintenance.sh

```diff
@@ -326,6 +365,30 @@ for FILENAME in `git ls-files`; do
     fi
 
 done
+
+rm -f doc/debug-messages.tmp1
+
+IMPORTANT_NUM=`gawk '{print $1}' doc/debug-messages.tmp2 | uniq -d`
```

Comment:
Please use camelCase for new variable names. There is no need to shout, even in shell, and newer source-maintenence.sh code usually follows Squid camelCase style.